docs/production-deployment/cloud/metrics/index.mdx

```diff
@@ -27,8 +27,18 @@ tags:
   - temporal-cloud
 ---
 
-Beyond the [metrics](/references/sdk-metrics) provided by the Temporal SDKs, some key metrics exist only in Temporal Cloud.
-You can use your own observability tool to query an endpoint and review Namespace metrics.
+Temporal offers two distinct sources of metrics: [Cloud/Server Metrics](/production-deployment/cloud/metrics/reference) and [SDK Metrics](/references/sdk-metrics).
+Each source provides options for levels of granularity and filtering, monitoring-tool integrations, and configuration.
+Before implementing observability for Temporal Cloud, decide what you need to measure for your use case. There are two primary use cases for metrics:
+- To measure the health and performance of Temporal-backed applications and key business processes.
+- To measure the health and performance of Temporal infrastructure and user provided infrastructure in the form of Temporal Workers and Temporal Clients.
+
+When measuring the performance of Temporal-backed applications and key business processes, you should rely on Temporal SDK metrics as a source of truth.
+This is because Temporal SDKs provide visibility from the perspective of your application, not from the perspective of the Temporal Service. 
+
+SDK metrics are for monitoring individual workers and your code's behavior.
```

Comment:
s/are for monitoring/monitor/

Same for the next line.

Removes passive voice, simplifies wording.